changelog/_unreleased/2021-12-16-remove-min-height-from-images-when-display-mode-is-not-cover.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Remove min height from images when display mode is not cover
+issue: NEXT-19312
+author: Marius Faber
+author_email: mariusfaber98@gmail.com 
+author_github: marius-faber
+---
+# Administration
+*  Removed min height from the image element and the image slider when display mode is not cover
+___
```

Comment:
Hi @marius-faber, thanks for your contribution. 🎉 

This divider can be removed when there are no further categories.